gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
Let's make this `-all` so the sources are available. Should be automatic by running `./gradlew wrapper --gradle-version 8.5`.

Reply:
Sources now could be attached by IDEA.

Reply:
K, then remove this stuff everywhere https://github.com/Goooler/gradle-play-publisher/blob/59b0a92a6a58318cd0cf54294328451b2ca7720a/testapp/build.gradle.kts#L39

And run `./gradlew wrapper --gradle-version 8.5` both top-level and in testapp.

Reply:
880950acffd7931c9203960c64b02ca099a1ae68